src/lv_draw/lv_draw_img.c

```diff
@@ -401,11 +401,7 @@ bool lv_img_color_format_is_chroma_keyed(lv_img_cf_t cf)
 
     switch(cf) {
         case LV_IMG_CF_TRUE_COLOR_CHROMA_KEYED:
-        case LV_IMG_CF_RAW_CHROMA_KEYED:
-        case LV_IMG_CF_INDEXED_1BIT:
-        case LV_IMG_CF_INDEXED_2BIT:
-        case LV_IMG_CF_INDEXED_4BIT:
-        case LV_IMG_CF_INDEXED_8BIT: is_chroma_keyed = true; break;
```

Comment:
These should be guarded with `LV_INDEXED_CHROMA`, shouldn't it?

Reply:
`LV_INDEXED_CHROMA` currently works by setting the transparency of any color in the palette equal to the chroma color to 0, but your suggestion seems like a clearer way of doing it, I'll change it.
https://github.com/littlevgl/lvgl/pull/1183/files#diff-dd44be118be5a3842537d69d2f2aa0d8R414-R423

Reply:
I have added this.

Reply:
However, the drawing algorithm needs to need that it's a chroma keyed or an alpha byte extended image data. 
See [here](https://github.com/littlevgl/lvgl/blob/3eb06252e9f1ec2e6e24938b7d6e7fbc6dbd4aaf/src/lv_draw/lv_draw_basic.c#L484-L504)

Reply:
I've just pushed a new commit to amend this! Please check the updated version.